src/runners/rfqm_worker_runner.ts

```diff
@@ -0,0 +1,97 @@
+/**
+ * Runs the RFQM MetaTransaction Consumer
+ */
+import { createMetricsRouter, MetricsService } from '@0x/api-utils';
+import { SQS } from 'aws-sdk';
+import express from 'express';
+import { Counter } from 'prom-client';
+
+import { ENABLE_PROMETHEUS_METRICS, PROMETHEUS_PORT, RFQM_META_TX_SQS_URL } from '../config';
+import { METRICS_PATH } from '../constants';
+import { getDBConnectionAsync } from '../db_connection';
+import { logger } from '../logger';
+import { RfqmService } from '../services/rfqm_service';
+import { SqsClient } from '../utils/sqs_client';
+import { SqsConsumer } from '../utils/sqs_consumer';
+
+import { buildRfqmServiceAsync } from './http_rfqm_service_runner';
+
+const RFQM_JOB_DEQUEUED = new Counter({
+    name: 'rfqm_job_dequeued',
+    help: 'An Rfqm Job was pulled from the queue',
+});
+
+const RFQM_JOB_SUCCEEDED = new Counter({
+    name: 'rfqm_job_succeeded',
+    help: 'An Rfqm Job succeeded',
+});
+
+process.on('uncaughtException', (err) => {
+    logger.error(err);
+    process.exit(1);
+});
+
+process.on('unhandledRejection', (err) => {
+    if (err) {
+        logger.error(err);
+    }
+});
+
+if (require.main === module) {
+    (async () => {
+        // Start the Metrics Server
+        startMetricsServer();
+
+        // Build dependencies
+        const connection = await getDBConnectionAsync();
```

Comment:
if there is a DB connection issue, will this LOC throw an error?

Reply:
I think this will be ok - this function throws an error only if the configs are wrong - and if the configs are wrong, the app will never work.